modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -276,6 +279,10 @@ private List<IRasSearchCriteria> getCriteria(
 			RasSearchCriteriaRunName runNameCriteria = new RasSearchCriteriaRunName(runName);
 			critList.add(runNameCriteria);
 		}
+		if(group != null && !group.isEmpty()){
```

Comment:
Really minor - the formatting in this `if` statement isn't consistent with the above conditions, would be great to keep things the same